.ci.yaml

```diff
@@ -109,6 +109,7 @@ platform_properties:
       os: Mac-12
       device_os: iOS-15.1
       xcode: 13a233
+      runtime_versions: ["ios-15-0"]
```

Comment:
That's not right either:
```
FormatException: Protobuf JSON decoding failed at: root["platform_properties"]["mac_ios"]["properties"]["runtime_versions"]. Expected String value
```

Reply:
There are a couple of issues here:

* These builders are using https://cs.opensource.google/flutter/recipes/+/main:recipe_modules/devicelab_osx_sdk/api.py which do not have the support for the runtime_versions.
* ci.yaml does not have support for runtime versions https://cs.opensource.google/flutter/infra/+/main:config/lib/ci_yaml/ci_yaml.star;l=182?q=xcode&ss=flutter%2Finfra